packages/flutter/lib/src/cupertino/text_selection.dart

```diff
@@ -347,12 +353,12 @@ class _CupertinoTextSelectionControls extends TextSelectionControls {
         return;
       }
 
-      if (items.isNotEmpty) {
-        items.add(onePhysicalPixelVerticalDivider);
-      }
-
       items.add(CupertinoButton(
-        child: Text(text, style: _kToolbarButtonFontStyle),
+        child: Text(
+          text,
+          overflow: TextOverflow.ellipsis,
```

Comment:
Just a FYI, this currently only work with word boundary. If for some reason the space is not enough, copy -> ... Paste -> ...    select all -> select ...

Reply:
I think somehow it's working ok, see this screenshot if I add a bunch of A's to Paste.  Is that what you mean?

<img width="204" alt="Screen Shot 2020-04-14 at 8 58 14 AM" src="https://user-images.githubusercontent.com/389558/79246506-2609e700-7e2e-11ea-8fe6-757874fe2d45.png">

Reply:
oh nvm it seems you are right.